webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js

```diff
@@ -14,6 +14,12 @@ var SignatureCorrelations = (function() {
                     if (response.count) {
                         $('li.correlations').show();
                     }
+                    if (response.errors) {
```

Comment:
The server doesn't include the key `errors` if there are no errors. So the code works as is. Arguably, perhaps it should. And then the if statement would need to be smarter.